z2/src/chain.rs

```diff
@@ -151,6 +152,22 @@ impl Chain {
         }
     }
 
+    pub fn get_forks(&self) -> Vec<Value> {
+        match self {
+            Chain::Zq2ProtoTestnet => vec![
+                json!({ "at_height": 0, "failed_scilla_call_from_gas_exempt_caller_causes_revert": false }),
+                // estimated: 2024-12-18 11:57:19Z
```

Comment:
Just a typo in the comment, should be `2024-12-18T11:57:19Z`
but let's activate the fork on the proto-testnet on Tuesday
and on the proto-mainnet 2 days later, on Thursday

Reply:
I've updated, thanks!